MANIFEST.in

```diff
@@ -9,3 +9,6 @@ recursive-include tests *
 
 # Include documentation in reStructuredText format
 recursive-include doc *
+
+# Required by setup.py script
+requirements.txt
```

Comment:
Shouldn't this be `include requirements.txt`?

Reply:
In fact, I would just write `include LICENSE requirements.txt` instead of the first few lines (`MANIFEST.in` doesn't need to be listed, IIRC)

Reply:
It was clearly time for bed when I wrote this... 🤦 

Fixed!